tutorials/primer/manipulate-data.md

```diff
@@ -414,9 +414,7 @@ A few examples:
 
 **Maths shards**
 
-The basic mathematical and comparison operators described above are built in as part of the mal language (core language on which Shards is based).
-
-However, Shards also provides a more extensive library of mathematical functions and comparison operators in form of various shards.
+If you need to go beyond basic mathematical computation and comparison you can use the extensive library of math and comparison shards included with the language.
```

Comment:
@satendrapalsingh i think we need a "dynamic of a shard program/app" section as it seems something hard to grasp even internally and anyway it's something necessary

Reply:
> @satendrapalsingh i think we need a "dynamic of a shard program/app" section as it seems something hard to grasp even internally and anyway it's something necessary

Yes, I agree - that would be best way to explain the dual nature of functions/shards. This will impact many sections, specifically "Elements of Shards" section, but this is the right way to do it.
  
I'm a bit disturbed that I'm missing something very obvious in my understanding of Shards, so starting a thread in docs channel for discussing the same.